src/Sarif/ExtensionMethods.cs

```diff
@@ -102,6 +102,29 @@ public static Message ToMessage(this string text)
             return new Message { Text = text };
         }
 
+        public static string ToAndPhrase(this List<string> words)
```

Comment:
>ToAndPhrase [](start = 29, length = 11)

In future, a bool to control whether discrete terms are quoted with apostrophes could be useful. Just something to keep in mind.